modules/bcf/app/views/bcf/issues/configure_non_members.html.erb

```diff
@@ -3,54 +3,48 @@
 
 <%= form_tag({ action: :configure_import }, multipart: true, method: :post) do %>
   <%= render 'import_options_hidden_fields' %>
-
-  <h3>Non project members found</h3>
+  <h3><%= t('bcf.bcf_xml.import.non_members_found') %></h3>
   <ul>
     <% @importer.aggregations.non_members.each do |user| %>
       <li><%= user.mail %> (<%= user.name %>)</li>
     <% end %>
   </ul>
   <% if @roles.any? %>
-    <p>What do you want to do?</p>
     <% if User.current.allowed_to?(:manage_members, @project) %>
-      <div class="form--field -full-width">
-        <div class="form--field-container">
-          <label class="option-label">
-            <input type="radio" name="import_options[non_members_action]" value="add" checked>
-            Add them as members to the project "<%= @project.name %>" with role
-            <% options = @roles.collect { |obj| [obj.name, obj.id] } %>
-            <%= select_tag 'import_options[non_members_add_role_ids][]', options_for_select(options),
-                           multiple: false,
-                           no_label: true,
-                           title:    l(:label_role_search),
-                           tabIndex: 0,
-                           class:    'form--select form--inline-select',
-                           id:       'non_members_add_role_ids' %>
-            (recommended)
-          </label>
+      <div class="form--field">
+        <input type="hidden" name="import_options[non_members_action]" value="add" checked>
+        <%= t('bcf.bcf_xml.import.add_as_members_with_role', project: @project.name) %>&nbsp;
+        <% options = @roles.collect { |obj| [obj.name, obj.id] } %>
+        <%= styled_select_tag 'import_options[non_members_add_role_ids][]',
+                              options_for_select(options),
+                              multiple: false,
+                              no_label: true,
+                              title:    l(:label_role_search),
+                              tabIndex: 0,
+                              class:    'form--select form--inline-select',
+                              container_class: '-slim',
+                              id:       'non_members_add_role_ids' %>
```

Comment:
You do not need to add the class `form--select` manually, the `styled_select_tag` takes care of this. Further the styles of the modifier `-slim` is overruled by the `form--inline-select` class. So one of the can be removed.